Yosemite/YosemiteTests/PointOfSale/PointOfSaleItemServiceTests.swift

```diff
@@ -283,4 +246,109 @@ final class PointOfSaleItemServiceTests: XCTestCase {
             XCTAssertEqual(error as? PointOfSaleItemServiceError, expectedError)
         }
     }
+
+    func test_providePointOfSaleVariationItems_formats_empty_prices_as_zero() async throws {
```

Comment:
Since we're supporting UK soon too, should we add new additional cases for `.GBP` here by Injecting a different `CurrencySettings` into the itemProvider?

Reply:
Since that's a responsibility of currency formatter, that seems like a fairly low-value test....

That would be a better argument if we checked whether a mock currency formatter was used, but I think we're probably OK here without further tests. Thanks for raising though!